README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Project status
 
-This project is stable, meaning the pace of new features is intentionally low. AKS Engine is maintained by teams who depend on it including [AKS Engine on Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-kubernetes-aks-engine-overview).
+This project is deprecated. Please consider using [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service/#overview) for managed Kubernetes or [Cluster API Provider Azure](https://github.com/kubernetes-sigs/cluster-api-provider-azure) for self-managed Kubernetes. This project will service major defects, but there are no planned new features. Kubernetes 1.24 will be the last version supported in AKS-Engine.
```

Comment:
I would prefer that we say "integrated into" or "added to" instead of "supported in".

Reply:
Updated